source/game_sa/CWanted.cpp

```diff
@@ -72,48 +61,51 @@ void CWanted::InitialiseStaticVariables()
 
 // Converted from thiscall void CWanted::UpdateWantedLevel(void) 0x561C90
 void CWanted::UpdateWantedLevel() {
-    //plugin::CallMethod<0x561C90, CWanted*>(this);
+    // TODO
+    // Find the bug. It sets 5 wanted stars instead of 6 when you use
+    // max wanted level cheat. Also you can get more than 4 stars in
+    // the begginning of the game
     m_nChaosLevel = std::min(m_nChaosLevel, MaximumChaosLevel);
 
     unsigned int wantedLevel = m_nWantedLevel;
 
     if (m_nChaosLevel > 4800) {
```

Comment:
```suggestion
    if (m_nChaosLevel > 4600) {
```
fixes the bug mentioned at the comment above